ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSource.mm

```diff
@@ -45,6 +58,7 @@ - (instancetype)initWithConfiguration:(CKTransactionalComponentDataSourceConfigu
     _announcer = [[CKTransactionalComponentDataSourceListenerAnnouncer alloc] init];
     _workQueue = dispatch_queue_create("org.componentkit.CKTransactionalComponentDataSource", DISPATCH_QUEUE_SERIAL);
     _pendingAsynchronousModifications = [NSMutableArray array];
+    _workThreadOverride = configuration.workThreadOverride;
```

Comment:
hmm, i'm kinda worried that then we're in the worst of both worlds — it seems weird that it comes from the configuration but changes are ignored.

regardless, for a temporary feature, why not.